transformer_engine/common/fused_attn/utils.h

```diff
@@ -152,11 +152,6 @@ class cudnnExecutionPlanManager {
     }
 
     ~cudnnExecutionPlanManager() {
```

Comment:
This is pretty ugly, so we should leave a comment to explain:
```suggestion
    ~cudnnExecutionPlanManager() {
      // Leak cuDNN handle to avoid strange errors from cudnnDestroy
```